msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -27,8 +27,7 @@ The rest of this guide walks you through these steps and show you how to preview
 To complete this tutorial, you need:
 
 * A Microsoft 365 Developer Program sandbox tenant.
-* Your sandbox tenant enrolled in *Microsoft 365 Targeted Releases*.
```

Comment:
Instead of removing this completely, how about making it optional? Even though Pages and Message Extension support are GA for M365/Outlook, upcoming features will still first roll out to the TR preview ring.